src/engraving/libmscore/textlinebase.cpp

```diff
@@ -451,6 +447,15 @@ void TextLineBaseSegment::layout()
     if (tl->lineVisible() || !score()->printing()) {
         pp1 = PointF(l, 0.0);
 
+        // Make sure baseline of text and line are propertly align (accounting fot line thickness)
+        if (isSingleBeginType() && (tl->beginTextPlace() == TextPlace::LEFT || tl->beginTextPlace() == TextPlace::AUTO)
```

Comment:
This code is impossible to understand/debug. It also has a lot of duplicated calls (e.g., isSingleBeginType() and tl->beginTextPlace())

Reply:
Ok I'll try to improve it, thanks

Reply:
Done, should be much more readable now 👍